tests/kernels/test_random_walk.py

```diff
@@ -0,0 +1,53 @@
+# Author: Henry Moss & Ryan-Rhys Griffiths
+"""
+Molecule kernels for Gaussian Process Regression implemented in GPflow.
+"""
+
+import sys
+
+import grakel
+import numpy.testing as npt
+import pandas as pd
+import pytest
+import tensorflow as tf
+from rdkit.Chem import MolFromSmiles
+from rdkit.Chem.rdmolops import GetAdjacencyMatrix
+
+from GP.kernel_modules.random_walk import RandomWalk
+
+sys.path.append('/Users/juliusschwartz/Mystuff/FlowMO')
```

Comment:
Any way to avoid this?

Reply:
I hope so xD This was absolutely a hack and I completely forgot that I'd left a hardcoded file path in here. I'd definitely get this PR rejected at work